src/helix-ui/elements/_hx-tooltip.less

```diff
@@ -0,0 +1,38 @@
+@import './_hx-element';
+@import '../styles/modules/mix-position.less';
+
+:host {
+    background-color: @gray-0;
+    border-color: @gray-300;
+    border-style: solid;
+    border-width: 1px;
+    box-shadow: 0px 3px 3px 0 @shadow-color;
+    color: initial;
+    display: none;
+    font-family: "Roboto", sans-serif;
+    font-size: 1rem;
+    font-style: normal;
+    font-weight: normal;
```

Comment:
What's with the numerous "reset" properties?

Reply:
They were added to address the issue of inheriting styles mentioned by Michael in a comment above where the tooltip was just inheriting the font styles of whatever it was in it i.e without the resets whenever the demo font-size is changed to 2rem with tooltip's font-size will be 2rem.